src/dummyData.js

```diff
@@ -0,0 +1,47 @@
+             { id: 1, profile_id: 7 }],
+    messages: [{ id: 0, sent_at: "2021-11-02T01:51:39+00:00", title: "Help with Homework?", read: true, message: "I need the answers to the assignment please.", inbox_id: 0, sender_id: 1 },
+               { id: 1, sent_at: "2021-11-02T01:51:39+00:00", title: "What's my grade?", read: true, message: "Hey Ms. Teacher can you tell me my grade?", inbox_id: 0, sender_id: 1 },
+               { id: 2, sent_at: "2021-11-02T01:51:39+00:00", title: "Going to need to cancel.", read: false, message: "My child has to miss the class", inbox_id: 1, sender_id: 2 },
+               { id: 3, sent_at: "2021-11-02T01:51:39+00:00", title: "Kid is sick", read: true, message: "Can we get a refund for this class?", inbox_id: 2, sender_id: 3 },
+               { id: 4, sent_at: "2021-11-02T01:51:39+00:00", title: "When is class?", read: false, message: "I noticed the time was funky and had to ask.", inbox_id: 1, sender_id: 5 },
+               { id: 5, sent_at: "2021-11-02T01:51:39+00:00", title: "Is this a yoga course?", read: false, message: "How is yoga and coding taught together?", inbox_id: 3, sender_id: 7 },
+               { id: 6, sent_at: "2021-11-02T01:51:39+00:00", title: "Where is my achievement?", read: false, message: "my achievement didn't pop up when I did class.", inbox_id: 0, sender_id: 1 }],
+    inbox: [{ id: 0, user_id: 0 },
+            { id: 1, user_id: 3 },
+            { id: 2, user_id: 8 }],
+    instructors: [{ id: 0, user_id: 9, rating: 0.5, bio: "lorem ipsum", approved: false, approved_by: "Joey" },
+                  { id: 1, user_id: 10, rating: 0.5, bio: "lorem ipsum", approved: false, approved_by: "Joey" },
+                  { id: 2, user_id: 11, rating: 0.5, bio: "lorem ipsum", approved: false, approved_by: "Joey" }],
+    enrollments: [{ id: 0, child_id: 0, course_id: 0, completed: false },
+                  { id: 1, child_id: 0, course_id: 0, completed: false },
+                  { id: 2, child_id: 0, course_id: 0, completed: false }],
+    instructor_list: [{ id: 0, instructor_id: 0, course_id: 0 },
+                      { id: 1, instructor_id: 0, course_id: 0 },
+                      { id: 2, instructor_id: 0, course_id: 0 }],
+    courses: [{ id: 0, size: 5, description: "lorem ipsup", subject: "lorem ipsum", prereq: "0" },
+              { id: 1, size: 5, description: "lorem ipsup", subject: "lorem ipsum", prereq: "0" },
+              { id: 2, size: 5, description: "lorem ipsup", subject: "lorem ipsum", prereq: "0" }],
+    schedules: [{ id: 0, course_id: 0, start_time: "5:00pm", start_date: "10-10-2010", end_date: "10-10-2010", end_time: "6:00pm", location: "Alabamma" },
+                { id: 1, course_id: 1, start_time: "6:00pm", start_date: "10-10-2010", end_date: "10-10-2010", end_time: "7:00pm", location: "Alabamma" },
+                { id: 2, course_id: 2, start_time: "7:00pm", start_date: "10-10-2010", end_date: "10-10-2010", end_time: "8:00pm", location: "Alabamma" }]
+}
```

Comment:
Looking good! Lint is throwing an error for a missing semicolon right here and GitHub is mad for no newline at the end of the file here too.

Reply:
Thanks for looking out! I just added the semicolon and pushed the changes.